portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -588,59 +588,78 @@ definition {
 		User.viewLoggedOutPG();
 	}
 
-	@description = "This is a use case for LPS-88924"
+	@description = "This is a use case for LPS-88924 using virtual instance"
```

Comment:
Please use the suffix format as it is in other test cases: " - Using virtual instances" Example in L#1897